github/repos_rules.go

```diff
@@ -97,6 +97,19 @@ type RequiredStatusChecksRuleParameters struct {
 	StrictRequiredStatusChecksPolicy bool                       `json:"strict_required_status_checks_policy"`
 }
 
+// RuleRequiredWorkflow represents the Workflow for the RequireWorkflowsRuleParameters object.
+type RuleRequiredWorkflow struct {
+	Path         string `json:"path"`
+	Ref          string `json:"ref,omitempty"`
+	RepositoryID *int64 `json:"repository_id,omitempty"`
+	Sha          string `json:"sha,omitempty"`
```

Comment:
Please add the pointers to the `omitempty` fields.

```suggestion
	Ref          *string `json:"ref,omitempty"`
	RepositoryID *int64 `json:"repository_id,omitempty"`
	Sha          *string `json:"sha,omitempty"`
```